indi-openogma/indi_openogma.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<driversList>
+  <device label="OpenOGMA Filter Wheel" manufacturer="OGMAVision">
```

Comment:
While not strictly required, I'd recommend you'd use a shorter name like **OPenOGMA EFW** for example.